### PR TITLE
rescue gem load errors so setup can run

### PR DESCRIPTION
**Pull request by indirect**
@segiddins r?

### CODE REVIEW

File: bin/rake [hunk 1]
```diff
@@ -4,7 +4,11 @@ require "rubygems"
 
 bundler_spec = Gem::Specification.load(File.expand_path("../../bundler.gemspec", __FILE__))
 bundler_spec.dependencies.each do |dep|
-  gem dep.name, dep.requirement.to_s
+  begin
+    gem dep.name, dep.requirement.to_s
+  rescue Gem::LoadError => e
+    puts "#{e.message} (#{e.class})"
```
Comment (segiddins) [inline bin/rake]: `$stderr.puts` ?

Comment (indirect) [inline bin/rake]: sure, I don't care ¯\_(ツ)_/¯

**Comment by segiddins** [comment]
@homu r+

**Comment by segiddins** [comment]
failure was a fluke.

**Comment by segiddins** [comment]
@homu retry

**Pull request merged**